### ISSUE TITLE
Support for Deutschlandticket

### ISSUE THREAD

**Issue by ghost** [ligi/PassAndroid#438]
While importing Deutschlandticket via .pkpass-file into PassAndroid I'm encountering an issue: it's taking forever to import the ticket.
How to import the ticket?

**Comment by chipotledonut** [comment]
Probably related to [this](https://github.com/ligi/PassAndroid/issues/431)

**Comment by lightonflux** [comment]
I too get the IO Exception described in #431 when importing a Deutschlandticket. 

Version 3.7.0 from F-Droid:
```
Invalid Pass
problem with temp file: 
java.io.IOException: Stream closed
```

Works without issue on Version 3.5.6 from Google Play.

**Comment by sandoxy** [comment]
Version 3.5.7 from F-Droid works, too.

**Comment by spaetz** [comment]
The stream closed errorbat least should be fixed in 3.7.1 https://github.com/ligi/PassAndroid/issues/444
